app/src/main/java/com/example/duccm/learningapp/LoginActivity.java

```diff
@@ -0,0 +1,58 @@
+package com.example.duccm.learningapp;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.text.TextUtils;
+import android.util.Patterns;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import java.util.regex.Pattern;
+
+public class LoginActivity extends AppCompatActivity {
+    Button buttonLogin;
+    private EditText editEmail, editPassword;
+    TextView textRegister;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login);
+        buttonLogin = (Button) findViewById(R.id.button_log_in);
+        textRegister = (TextView) findViewById(R.id.text_register);
+        editEmail = (EditText) findViewById(R.id.edit_email);
+        editPassword = (EditText) findViewById(R.id.edit_password);
+
+        buttonLogin.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                if (isValidEmail(editEmail.getText().toString()) &&
```

Comment:
nhiều view cùng có onclick thì em nên implements vào Activity rồi setOnClickListener(this)
bên trong onClick(View v) sẽ switch case theo id của view